### PR TITLE
Sdks 2524 update authenticator sample app

### PR DESCRIPTION
**Pull request by vahancouver**
[SDKS-2524](https://bugster.forgerock.org/jira/browse/SDKS-2524)

Add deep link support in Authenticator sample app

### CODE REVIEW

File: SampleApps/FRAuthenticatorExample/FRAuthenticatorExample/Info.plist [hunk 1]
```diff
@@ -16,6 +16,29 @@
 	<string>$(PRODUCT_BUNDLE_PACKAGE_TYPE)</string>
 	<key>CFBundleShortVersionString</key>
 	<string>1.0</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
```
Comment (rodrigoareis) [inline SampleApps/FRAuthenticatorExample/FRAuthenticatorExample/Info.plist:21]: Missing new `mfauth` scheme.

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Overall changes looks good to me. Just a minor change required.

File: SampleApps/FRAuthenticatorExample/FRAuthenticatorExample/Info.plist (hunk 1 above)
Comment (vahancouver) [reply SampleApps/FRAuthenticatorExample/FRAuthenticatorExample/Info.plist]: Added

**Review by rodrigoareis — APPROVED** [review]
LGTM

**Review by spetrov — APPROVED** [review]
LGTM! 👍🏻

**Pull request merged**